src/reference/interactive/index.pug

```diff
@@ -86,62 +88,67 @@ block reference
-
-    p Interactive developers can use buttons to allow participants to contribute to votes, cause actions to happen or even control in-game entities directly.
+        li `Down` event is sent when a button is pressed by a participant
+        li `Up` event is sent when a button is released by a participant
+    
+    .figure
+        img(src='/img/reference/interactive/button.png' alt='Sample button control which costs the participant one spark if they click on it.')
+    p.caption Sample button control which costs the participant one spark if they click on it.
+    
+    p You can use buttons to enable participants to vote, cause in-game actions to happen, or control in-game entities.
 
-    p Buttons have a number of properties which can be edited in both the Interactive Studio and from a Game Client.
+    p Buttons are customizable. These button properties can be edited from both the Interactive Studio and a Game Client:
     ol(style='list-style-type: decimal')
-        li Displayed text
-        li Spark Cost
-        li Progress Bar Width - Displayed at the bottom of a button.
-        li Disabled State - Disabled buttons cannot be interacted with.
-        li Cooldown Duration - Prevents interaction until it expires.
+        li Display text
+        li Spark cost (For more inof about sparks, go to #[a(href='#what-are-sparks') What are Sparks?]
+        li Width of the progress bar, which is displayed at the bottom of a button
+        li State of the button - Enable or disable them, disabled buttons cannot be interacted with
+        li Cooldown duration - Prevents interaction until it expires
 
     h5#what-are-sparks What are Sparks?
 
-    p Sparks are Beam's virtual currency. Participants earn them while watching or participating in streams everywhere on Beam.
+    p Sparks are Beam's virtual currency. Participants earn them while watching or participating in live video streams everywhere on Beam.
 
     h5#spark-transactions Spark Transactions
     .figure
         img(src='/img/reference/interactive/TransactionLifecycle.svg' alt='Diagram of a transaction\'s life cycle')
     p.caption Diagram of a spark transaction's life cycle, showing its transition betwen states.
 
-    p When a button with a spark cost is pressed, it creates a transaction. To deduct sparks from a participant a #[a(href='#the-game-client') Game Client] must #[code capture] the transaction. If a transaction remains uncaptured for 5 minutes, it expires, and the cost associated with the transaction is not deducted from the user's spark balance.
+    p When a button with a spark cost is clicked, it creates a transaction. To deduct sparks from a participant, a #[a(href='#the-game-client') Game Client] must #[code capture] the transaction. If a transaction remains uncaptured for 5 minutes, it automatically expires, and the cost associated with the transaction is not deducted from the participant's spark balance.
```

Comment:
"activated"? echoing comments about keyboard and controller support from previous comments.